src/imitate/create-imitation.js

```diff
@@ -11,7 +11,7 @@ export default (original, names) => {
       return original
     } else {
       // TODO: this will become src/function/create and include parent reference instead of name joining here
-      return tdFunction(names.join('') || '(anonymous function)')
+      return tdFunction(names.map(String).join('') || '(anonymous function)')
```

Comment:
Can you please use `_.map` for this? We're using lodash for all of our enumerable methods everywhere else, and there's a tacit (but not explicit) support for ancient browsers / IE etc that won't have `Array.prototype.map` defined.